app/javascript/components/Waybill/WaybillTable.tsx

```diff
@@ -39,7 +43,7 @@ const WaybillTable: React.FC<WaybillTableProps> = (props: WaybillTableProps) =>
                   <StyledTableCell>No data yet ...</StyledTableCell>
                 </TableRow>
               )
-              : waybills.map((waybill) => (
+              : data.map((waybill) => (
```

Comment:
just personal opinion, but name "data" is not showing for me whats inside it. Maybe better naming would be waybillsData or smth